### PR TITLE
Move pay houses to lua

### PR DESCRIPTION
**Pull request by ramon-bernardo**
### Pull Request Prelude
- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- Move pay houses to startup server
- Allow pay for houses calling Game.payHouses(rentPeriod)

I need to test this changes.

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by MillhioreBT** [comment]
I honestly don't like the use of `goto`, I prefer the double loop trick with for-repeat.

```lua
for k, v in pairs({}) do
    repeat
        -- code ...
        if not x then break end -- continue
        -- code ...
        if not h then break end -- continue
    until true
end
```

**Comment by ramon-bernardo** [comment]
@MillhioreBT I liked repeat

File: data/lib/core/game.lua [hunk 1]
```diff
@@ -188,6 +188,114 @@ function Game.getUnpromotedVocations()
...
+	end
+
+	function Game.payHouses(rentPeriod)
+		if rentPeriod == RENTPERIOD_NEVER then
+			return
+		end
+
+		local currentTime = os.time()
+		local paidUntil = currentTime + getHousePaidUntil(rentPeriod)
+		local nameRentPeriod = getHouseRentPeriod(rentPeriod)
```
Comment (MillhioreBT) [inline data/lib/core/game.lua:239]: `getHouseRentPeriod` no exists.

**Review by MillhioreBT — CHANGES_REQUESTED** [review]

File: data/lib/core/game.lua [hunk 2]
```diff
@@ -188,6 +188,114 @@ function Game.getUnpromotedVocations()
...
+
+				local playerBalance = player:getBankBalance()
+				if playerBalance >= rent then
+					player:setBankBalance(playerBalance - rent)
+					house:setPaidUntil(paidUntil)
+					house:setPayRentWarnings(0)
+				elseif payRentWarnings < 7 then
+					local daysLeft = 7 - payRentWarnings
+					
+					local stampedLetter = Item(ITEM_LETTER_STAMPED)
```
Comment (MillhioreBT) [inline data/lib/core/game.lua:279]: The Item constructor is not used to create items, it is used to obtain a temporary item in the environment
you should use Game.creteItem

**Review by MillhioreBT — CHANGES_REQUESTED** [review]

File: data/lib/core/game.lua (hunk 2 above)
Comment (ramon-bernardo) [reply data/lib/core/game.lua]: Thanks

**Pull request closed without merging**